features/git-rename-branch/non_feature_branch/with_remote_origin/with_tracking_branch/with_open_changes.feature

```diff
@@ -0,0 +1,44 @@
+Feature: git rename-branch: renaming a non-feature branch with a tracking branch (with open changes)
+
+  As a developer with a poorly named non-feature branch
+  I want to be able to rename it safely in one easy step
+  So that I can stay organized and remain productive
+
+
+  Background:
+    Given I have branches named "qa" and "production"
+    And my non-feature branches are configured as "qa" and "production"
+    And the following commits exist in my repository
+      | BRANCH     | LOCATION         | MESSAGE           |
+      | main       | local and remote | main commit       |
+      | production | local and remote | production commit |
+      | qa         | local and remote | qa commit         |
+    And I am on the "production" branch
+    And I have an uncommitted file
+
+
+  Scenario: error when trying to rename
+    When I run `git rename-branch production renamed-production`
+    Then it runs no Git commands
+    And I get the error "The branch 'production' is not a feature branch. Run 'git rename-branch production renamed-production -f' to force the rename, then reconfigure git-town on any other clones of this repo."
```

Comment:
The name for this tool currently is `Git Town`, not `git-town`. I'm happy either way, we just should be consistent.